pkg/solana/txm/txm.go

```diff
@@ -748,14 +748,11 @@ func (txm *Txm) Enqueue(ctx context.Context, accountID string, tx *solanaGo.Tran
 	// Perform compute unit limit estimation after storing transaction
 	// If error found during simulation, transaction should be in storage to mark accordingly
 	if cfg.EstimateComputeUnitLimit {
```

Comment:
So simulating and failing a Tx if it exceeds its ComputeUnitLimit is conditional on this field being set to true.
How do we ensure this field is set to true always for CCIP?